### PR TITLE
[14.0][MIG] base_delivery_carrier_label: port pr from 12.0 to 14.0

### PR DESCRIPTION
**Pull request by sonhd91**
* [[12.0][RFR] base_delivery_carrier_label: show options as tags](https://github.com/OCA/delivery-carrier/pull/345)
* [[12.0][RFR] carrier label tests: extract Test classes from Case classes for easier overriding](https://github.com/OCA/delivery-carrier/pull/352)
* [[12.0][FIX] base_delivery_carrier_label: form views usability](https://github.com/OCA/delivery-carrier/pull/391)

### CODE REVIEW

File: base_delivery_carrier_label/tests/carrier_label_case.py [hunk 1]
```diff
@@ -13,6 +13,10 @@ def setUp(self, *args, **kwargs):
         super().setUp(*args, **kwargs)
         self._create_order_picking()
 
+    @property
+    def transfer_in_setup(self):
```
Comment (leemannd) [inline base_delivery_carrier_label/tests/carrier_label_case.py:17]: I'm not sure to see the added value

**Review by leemannd — APPROVED** [review]
I'm wondering a bit about the modification on the test, but base PR seems good

File: base_delivery_carrier_label/tests/carrier_label_case.py (hunk 1 above)
Comment (sonhd91) [reply base_delivery_carrier_label/tests/carrier_label_case.py:17]: As mentioned in this [PR](https://github.com/OCA/delivery-carrier/pull/352), this one allow to override whether to transfer the picking in setUp()

**Comment by sonhd91** [comment]
> I'm wondering a bit about the modification on the test, but base PR seems good

It's add test case for option tag color

**Review by ajaniszewska-dev — APPROVED** [review]

**Review by ivantodorovich — APPROVED** [review]

**Review by sebalix — APPROVED** [review]

**Pull request merged**